src/app/content/components/ContentLink.tsx

```diff
@@ -95,14 +96,30 @@ export const ContentLink = (props: React.PropsWithChildren<Props>) => {
         navigate(navigationMatch, options);
       }
     }}
+    onKeyDown={(e) => onKeyDown && onKeyDown(e, async() => {
```

Comment:
You can actually write this execute-if-defined as `onKeyDown?.(e, async() => {`
An similarly for the `onClick` below.